src/volttron/client/known_identities.py

```diff
@@ -53,10 +53,10 @@
 PLATFORM_MARKET_SERVICE = "platform.market"
 
 ROUTER = ""
-CONTROL = "platform.control"
+CONTROL = "volttron.platform.control"
```

Comment:
To make it consistent with server side object.  The service name is volttron.services.control so it made sense to me for a volttron.platform.control to make all of them have volttron.platform rather than leave it as platform.